src/components/tag/tag.styles.ts

```diff
@@ -11,99 +11,95 @@ export default css`
   .tag {
     display: flex;
     align-items: center;
-    border: solid 1px;
+    border: none;
+    border-radius: var(--ts-border-radius-x-small);
     line-height: 1;
     white-space: nowrap;
     user-select: none;
   }
 
+  .tag:active > sl-icon-button {
+    color: var(--sl-color-neutral-900);
+  }
+
   .tag__remove::part(base) {
-    color: inherit;
+    color: var(--sl-color-neutral-700);
     padding: 0;
   }
 
   /*
    * Variant modifiers
    */
 
-  .tag--primary {
-    background-color: var(--sl-color-primary-50);
-    border-color: var(--sl-color-primary-200);
+  .tag--blue {
```

Comment:
I'm curious about why we're switching to presentational classes instead of semantic ones. Wouldn't that encourage people to use them in arbitrary places?

Reply:
The design team went back & forth on this a bit. We wanted to add three new decorative (non-semantic) colors, which seemed inconsistent to have alongside the existing 5 semantic naming for the variants, and we weren't sure that the `primary` (blue) variant makes sense for us anyway, so we landed on just switching to presentational naming & just adding guidelines in Figma and the doc site (to do later) for generally reserving Yellow, Red, and Green for semantic uses. But after seeing that we're using the `sl-tag` in a few places already I thought it wouldn't be a bad thing to keep the semantically named variants as well, so that you could use either `danger` or `red` to create a red tag. Open to your thoughts on how to update though, if you think this is a bad idea!